certbot-nginx/certbot_nginx/_internal/configurator.py

```diff
@@ -693,6 +693,7 @@ def _get_snakeoil_paths(self) -> Tuple[str, str]:
         le_key = crypto_util.generate_key(
             key_size=1024, key_dir=tmp_dir, keyname="key.pem",
             strict_permissions=self.config.strict_permissions)
+        assert le_key.file is not None
```

Comment:
It's to satisfy `mypy`. The value must be set because `key_dir` was set.

We use this pattern elsewhere in Certbot, especially during Adrien's typing work.

Do you prefer another approach here?

Reply:
Ah! OK. I didn't realize it was for mypy. I personally think this is fine then.